ring/partition_ring_model.go

```diff
@@ -94,6 +94,34 @@ func (m *PartitionRingDesc) partitionByToken() map[Token]int32 {
 	return out
 }
 
+// CountTokens returns the summed token distance of all tokens in each partition.
+func (m *PartitionRingDesc) countTokens() map[int32]int64 {
```

Comment:
Chose this name for consistency with its corresponding operation on the instance ring: https://github.com/grafana/dskit/blob/9935aca9d266be5de05450afd8278d99637a24b6/ring/ring.go#L625